src/modules/api/user/link/link.controller.ts

```diff
@@ -53,4 +59,23 @@ export class LinkController {
       return { status: 'NOT OK', error: error.message };
     }
   }
+  @Put(':id')
+  async editLink(
+    @Param('userId') userId: string,
+    @Param('id') linkId: string,
+    @Body() linkData: ILink,
+  ): Promise<IResult<LinkDTO>> {
+    try {
+      const result = await this.linkService.editUserLink(linkId, linkData);
+      const updatedLink = new LinkDTO(result);
+      return {
+        status: 'OK',
+      } as IResult<LinkDTO>;
```

Comment:
Send the `updatedLink` as `result`

Reply:
Is the ststus not sufficient like however we will use get to get the new data right?

Reply:
Well, Generally we do return the updated or deleted objects. We can use it in dialogues or something to show that they are updated and the action is successful.